airgun/widgets.py

```diff
@@ -2238,3 +2240,43 @@ def items(self):
 
     def toggle(self, value):
         self.browser.click(self.ITEM.format(value))
+
+
+class BaseMultiSelect(BaseSelect, Dropdown):
+    """Represents the Patternfly Multi Select.
+
+    https://www.patternfly.org/v4/documentation/react/components/select#multiple
+    """
+
+    BUTTON_LOCATOR = './/button'
```

Comment:
This is very straight locator, dont we have a unique identifier for that button, so if tomorrow one more button gets added to the view, it should not fail ?